components/fee-button.js

```diff
@@ -118,7 +118,7 @@ export default function FeeButton ({ ChildButton = SubmitButton, variant, text,
     ? 'free'
     : total > 1
       ? numWithUnits(total, { abbreviate: false, format: true })
-      : undefined
+      : 'free'
```

Comment:
this is not helpful

Reply:
This does indeed fix #908. It no longer overlays the button with `1 sat` even though editing is free.

However, actions that require 1 sat like comments are now always shown as free, even when one does have enough sats in their wallet.

Using `total >= 1` instead of `total > 1` above would fix that but then we would show `1 sat` inside the button which we don't want. We were using `undefined` here because of that.

So this should fix #908 and prevent a regression:

```suggestion
      : total === 1 ? undefined : 'free'
```

It's not ideal code (too many ternary operators) but imo it's okay to get #908 fixed.